### PR TITLE
Do introduce

### LINKED ISSUE #1: Do introduce myself
i will do intoduce my self

- [ ] hobby
- [ ] what i did today
- [ ] residence
- [ ] lunch menu tomorrow

### PR DESCRIPTION
**Pull request by pyozz**
resolve #1

### CODE REVIEW

**Review by Myugaa — CHANGES_REQUESTED** [review]
Please fix the 4th line (-ksd...)

File: introduction.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
 	 2. 이병욱
 	 
 	 3. 표승연
+         - tennis
+         - study hard
```
Comment (youzysu) [inline introduction.html:12]: 정말인가요?

File: introduction.html [hunk 2]
```diff
@@ -8,6 +8,10 @@
...
+         - Seoul
+         - ksdfjdlksfjasdklf
```
Comment (youzysu) [inline introduction.html:14]: 이게 무엇이죠?

**Review by youzysu — CHANGES_REQUESTED** [review]

**Comment by Myugaa** [comment]
Merged!

**Pull request merged**